update.php

```diff
@@ -256,3 +256,49 @@ function update_1288() {
 
 	return Update::SUCCESS;
 }
+
+	
+// Post-update script of PR 5751
+function update_1293() {
```

Comment:
Code standards: Please add a new line before function opening brace.

Reply:
It's weird to see this comment while all other functions in this file have the same style.

Reply:
https://github.com/friendica/friendica/blob/develop/update.php

Reply:
Please rest assured I'm not singling you out. This file has been created before I started badgering people with code standards, and not all files have been retroactively fixed in that regard. However, any added code is scrutinized equally, regardless of the rest of the file (except for quote style that must match the most popular style in the function/file)

Reply:
And at some point, without realizing, you start to write correct code :-)

Reply:
A little delay, I didn't mean it like this. I also think having good code standards is neccesary. Otherwise you'll get a mik-mak of several styles and no one wants to contribute because nobody understands anymore.

Moreover, when this PR is I'll create another PR where I'll update update.php towards code standards :)

Reply:
That's the spirit!